zerver/tests/test_digest.py

```diff
@@ -325,6 +333,16 @@ def test_twelve_hour_exemption(self) -> None:
 
         self.assertEqual(queue_mock.call_count, 0)
 
+    @override_settings(SEND_DIGEST_EMAILS=True)
+    @override_settings(SYSTEM_ONLY_REALMS=["zulipinternal"])
+    def test_enqueue_emails(self) -> None:
+        # code coverage
+        Realm.objects.update(digest_emails_enabled=True)
+        Realm.objects.update(deactivated=False)
+        Realm.objects.update(digest_weekday=timezone_now().weekday())
```

Comment:
We try to avoid directly modifying database tables in tests; use `do_set_realm_property` for these changes.

(We've been meaning to do a linter sweep for this sort of thing in tests).

Reply:
DONE: thanks! for the tip. I also added a proper correct-execution test.